src/search/search-box/search-box.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import { SearchBox } from './search-box';
+import { SearchContextProvider } from '../context/search.context';
+
+const meta: Meta<typeof SearchBox> = {
+  title: 'SEARCH/SearchBox',
+  component: SearchBox,
+  tags: ['autodocs'],
+  args: {
+    /**
+     * For the component properties which has `ReactNode` type set the default value as empty string
+     * because the Storybook doesn't allow editing the `ReactNode` type properties in the controls.
+     */
+  },
+};
+
+const account_id = 6413;
+const auth_key = "bcvpynhij980k0y1";
+const domain_key = "pacifichome";
+
+export default meta;
+
+export type Story = StoryObj<typeof SearchBox>;
+
+export const Basic: Story = {
+  render: (args) => {
+    return (
+      <SearchContextProvider>
+        <SearchBox {...args} />
```

Comment:
Could you explain how I can interact with it as an end user? How can I access the response to work with it? Maybe I don't understand this due to being out of knowledge of the full things with sdk.

Reply:
I will add to the example and story the part where one uses the SearchContext to get the search response dataand render that. My vision was to use the searchbox to retreive and store the results in the Context so those results can be used in other component / parts of the app.